README.md

```diff
@@ -89,7 +89,7 @@ api.getTheaters(function (error, theaters) {
 #### `showtimes.getMovies(callback)`
 * `callback` &mdash; Callback function to run after generating a standardized response of movies.
   * `error` &mdash; Any error messages that were found when doing a query. Null if all is good.
-  * `theaters` &mdash; Standardized response of theaters.
+  * `theaters` &mdash; Standardized response of movies.
```

Comment:
Oh, good catch. `theaters` should also be `movies` as well, if you want to fix that too in this PR.